libs/dh/core/shell/src/lib/shell/shell.component.scss

```diff
@@ -14,10 +14,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@use '@energinet-datahub/watt/utils' as watt;
+
 :host {
   display: block;
 }
 
 .app-actions {
   margin-left: auto;
 }
+
+.sidenav {
+  display: flex;
+  justify-content: center;
+}
+
+.logo {
+  &--toolbar {
+    @include watt.media('>=Large') {
```

Comment:
praise: I didn't know using `>=` was also an option. Nice.

suggestion: Can you add it to the Breakpoint docs?